enclave_manager/avalon_enclave_manager/kme/kme_enclave_manager.py

```diff
@@ -152,21 +153,44 @@ def _start_kme_listener(self):
     def GetUniqueVerificationKey(self, **params):
         """
         """
+        workload_id = "_kme-uid"
+        verification_key_nonce = params["nonce"]
+        in_data = json.dumps({"nonce": verification_key_nonce})
+        wo_req = construct_wo_req(in_data, workload_id, self.encryption_key)
+
+        # @TODO : Trusted implementation to be integrated
+        # wo_response = self._execute_work_order(json.dumps(wo_req))
+
         return ""
 
 # -----------------------------------------------------------------
 
     def RegisterWorkOrderProcessor(self, **params):
         """
         """
+        workload_id = "_kme-reg"
+        attestation_report = params["attestation_report"]
+        in_data = json.dumps({"attestation_report": attestation_report})
+        wo_req = construct_wo_req(in_data, workload_id, self.encryption_key)
+
+        # @TODO : Trusted implementation to be integrated
```

Comment:
FYI registration needs one more argument "verification_key" go it in first step(GetUniqueId)